src/components/ArticleLabel/Styles.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
+
+import {fonts} from 'styles';
+
+export const Container = styled.div`
+  width: 100%;
+  padding: 15px;
+`;
+
+export const Box = styled.div`
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  margin: 10px 0px 15px 0px;
+  width: 100%;
+`;
+export const BoxLeft = styled.div`
```

Comment:
Again, this `BoxLeft` is too generic,

Maybe we can try something like: `LogoContainer` etc..
<img width="379" alt="image" src="https://github.com/thenewboston-developers/thenewboston-Frontend/assets/52558124/d7556e81-10fd-478d-b7bc-b7a16564c324">

Reply:
That's right, I accept